moped-database/migrations/1699408061245_create_table_public_moped_user_events/up.sql

```diff
@@ -1,10 +1,38 @@
+ALTER TABLE "public"."moped_users"
+    ADD COLUMN "last_seen_date" timestamptz NULL;
+
+COMMENT ON COLUMN moped_users.last_seen_date IS 'Tracks the last time a user loaded the Moped app in their browser. This value is set by the set_last_seen_date function. This value is not 100% reliable because it is updated by an API call that can be blocked by the client.';
+
+
 CREATE TABLE "public"."moped_user_events" (
     "id" serial NOT NULL,
     "user_id" integer NOT NULL,
-    "description" text NOT NULL,
+    "event_name" text NOT NULL CHECK (event_name in('app_load')),
```

Comment:
if we want to support additional event types, we would add them to this check constraint